src/Feedback.js

```diff
@@ -0,0 +1,145 @@
+import Typography from "@material-ui/core/Typography";
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+
+var radio = "No"
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: 'flex',
+    },
+    content: {
+        flexGrow: 1,
+        padding: theme.spacing(3),
+    },
+    feedback_form: {   
+    },
+}));
+
+function getInputData() {
+    // get data from html form
+    let dataToPost = new FormData(); //formdata API
+
+    //fill name attributes to corresponding values
+    dataToPost.append("entry.2006203568", document.querySelector("#name").value);
+    dataToPost.append("entry.217739706", document.querySelector("#email").value);
+    dataToPost.append("entry.324300712", radio);
+    dataToPost.append("entry.667358521", document.querySelector("#q1").value);
+    dataToPost.append("entry.1924495588", document.querySelector("#q2").value);
+    dataToPost.append("entry.870162535", document.querySelector("#q3").value);
+    dataToPost.append("entry.952911278", document.querySelector("#q4").value);
+    dataToPost.append("entry.113005119", document.querySelector("#q5").value);
+    dataToPost.append("entry.1558581568", document.querySelector("#q6").value);
+
+    return dataToPost;
+}
+
+function eraseInput(){
+    // clear input fields after submission
+    document.querySelector("#name").value = "";
+    document.querySelector("#email").value = "";
+    document.querySelector("#q1").value = "";
+    document.querySelector("#q2").value = "";
+    document.querySelector("#q3").value = "";
+    document.querySelector("#q4").value = "";
+    document.querySelector("#q5").value = "";
+    document.querySelector("#q6").value = "";
+
+}
+
+function on_submit(e) {
+    // prevents redirect
+    e.preventDefault();
+    let url = "https://docs.google.com/forms/d/e/1FAIpQLSdKXxrigrsShBGqU_hJhwR-WzFFHNeefLGZT81R13Mgx4nfXg/formResponse";
+
+    // collecting and sending the data to the google form
+    fetch(url,{
+        method: "POST",
+        mode: "no-cors",
+        header:{
+            'Content-Type': 'application/json'
+            },
+        body: getInputData()
+    })
+
+    // after submitted success
+    .then(data=>{
+        let success_msg = document.querySelector("#success");
+        success_msg.style.display = "block";
+        setTimeout( () => {success_msg.style.display = "none"}, 30000);
+        eraseInput()
+    })
+    // if error on post
+    .catch(err=>console.error(err)); //promise based
+}
+
+
+function Feedback(props) {
+    const { strings } = props;
+    const classes = useStyles();
+    const [value, setValue] = React.useState('No');
```

Comment:
lol, i had to look up what this does. it basically makes it so that we can use a bit of react state functionality without using a React class 🙄 fine for now since it's working, but we could probably clean up the radio-button handling next semester. also it's a bit odd to keep track of the radio value through `radio` when we're tracking it through the `React.useState` and its `setValue()` method.

Reply:
Yeah ... this one is also on me. Couldn't figure out a way to access the state for the react function so I created the variable radio to track it's state.

Reply:
+1 on this being weird, I am not a react expert so maybe it's longer than it would for someone better at this but it took me a minute to figure out what was tracking what state. That being said, if it's fine for now with y'all and you want to fix it next semester might be worth it to make a GH issue or task or something to specify that it needs to happen. It would also be good to add a comment or something probably.

Reply:
+1 to tracking this with a Github issue and a comment for now